frontend/packages/knative-plugin/integration-tests/features/eventing/create-event-sink.feature

```diff
@@ -0,0 +1,68 @@
+@knative-eventing
+Feature: Create event sinks
+              As a user, I want to create event sinks
+
+        Background:
+            Given user has installed Red Hat Integration - Camel K Operator
+              And user has created or selected namespace "aut-event-sink"
+              And user has created channel "kn-channel"
+              And user is at Add page
+
+
+        @regression @odc-6390
+        Scenario: Create event sink from Add page: KES-01-TC01
+             When user clicks on Event Sink card
+              And user clicks on "Kafka Sink" card
```

Comment:
Might be better to use a different Event Sink. In future we will have a sink of kind `KafkaSink` whereas the one you have selected is of kind `Kamelet`

Reply:
Updated